src/internal/snapshot/server_internal_test.go

```diff
@@ -50,6 +54,50 @@ func TestListSnapshot(t *testing.T) {
 	require.NoDiff(t, allRows[:3], sinceSecondSnapshot, []cmp.Option{protocmp.Transform()})
 }
 
+func TestInspectSnapshot(t *testing.T) {
+	ctx := pctx.TestContext(t)
+	c := pachd.NewTestPachd(t)
+	createResp, err := c.CreateSnapshot(ctx, &snapshot.CreateSnapshotRequest{})
+	if err != nil {
+		t.Fatalf("create snapshot RPC: %v", err)
+	}
+	inspectResp, err := c.InspectSnapshot(ctx, &snapshot.InspectSnapshotRequest{Id: createResp.Id})
+	if err != nil {
+		t.Fatalf("inspect snapshot RPC: %v", err)
+	}
+	info := inspectResp.Info
```

Comment:
https://go.dev/wiki/TestComments#compare-full-structures
Create a `got` and `want` structure, then `require.NoDiff()` or `if diff := cmp.Diff(want, got); diff != "" { t.Errorf("info (-want +got):\n%s", diff) }`.